src/FSharp.Control.TaskSeq/Utils.fsi

```diff
@@ -6,10 +6,10 @@ open System.Threading.Tasks.Sources
 
 [<AutoOpen>]
 module ValueTaskExtensions =
-    type System.Threading.Tasks.ValueTask with
+    type ValueTask with
 
         /// (Extension member) Gets a task that has already completed successfully.
-        static member inline CompletedTask: System.Threading.Tasks.ValueTask
```

Comment:
is there a specific reason this can't live in the module? Might be worth a comment...

Reply:
It deliberately shadows `CompletedTask`, that exists by default in later versions of .NET. This way, people that use the later version of .NET will get the build-in version, but users that want to use it and are bound to .NET Standard or other older versions, can use this.

Reply:
and that would not work if it was in the `module` (even if the naming would be inconsistent) ?
I will put a comment beside it as to this purpose
let me know if you want me to move it to the `module` (but given I'm not sure it'll work that way I'm definitely not forcing it)

Reply:
No no, it must stay as an extension member on the type.

Reply:
Cool - will just add a comment noting that and it's raison d'

Reply:
https://github.com/fsprojects/FSharp.Control.TaskSeq/pull/249/commits/b88153a26cb81f4ba97cdeb659e1004b801dcaee

Thanks for the details - I think I confused myself, but the verbosity of the existing comments didnt help, and I shifted the contextual comments to the fsi and the impl detail to the fs

(But let me know if it's an overreach)